zipkin-lens/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "extract-messages": "extract-messages -l=en,zh-cn --flat -o src/translations src/**/messages.js",
+    "extract-messages": "extract-messages -l=en,zh-cn,es --flat -o src/translations src/**/messages.js",
```

Comment:
I realized you must have manually set up the translation files - sorry for not documenting this command yet! I will do so but after addressing some current plans I have for the i18n setup.